src/website/app/pages/ComponentDoc/index.js

```diff
@@ -57,18 +59,40 @@ type BestPractice = {
 
 type Theme = (theme: Object) => Object;
 
-const StyledDocHeading = createStyledComponent(Heading, ({ theme }) => ({
-  marginBottom: 0,
-  paddingTop: getNormalizedValue(pxToEm(62), theme.SiteHeading_fontSize_2),
+const styles = {
+  docHeading: ({ theme }) => ({
+    marginBottom: 0,
+    paddingTop: getNormalizedValue(pxToEm(62), theme.SiteHeading_fontSize_2),
 
-  [theme.bp_moreSpacious]: {
-    fontSize: theme.SiteHeading_fontSize_2_wide,
-    paddingTop: getNormalizedValue(
-      pxToEm(80),
-      theme.SiteHeading_fontSize_2_wide
-    )
-  }
-})).withProps({ level: 2 });
+    [theme.bp_moreSpacious]: {
+      fontSize: theme.SiteHeading_fontSize_2_wide,
+      paddingTop: getNormalizedValue(
+        pxToEm(80),
+        theme.SiteHeading_fontSize_2_wide
+      )
+    }
+  }),
+  importSyntax: ({ theme }) => ({
```

Comment:
You called it `importSyntax` here - let's do that everywhere....

Reply:
Actually, can/should this make use of the existing `Callout` component instead?

Reply:
yep, @kylegach and I discussed that yesterday and I'm implementing it.